common.mk

```diff
@@ -33,6 +33,16 @@ ifneq ($(shell python -c "import wheel as w; \
 $(error Looks like the `wheel` package is outdated or missing. See README for instructions on how to fix this.)
 endif
 
+# This check should not occur within CI environments, where AWS Credentials might not be supplied
+ifneq ($(shell python -c "import os, sys; \
+                          import boto3 as b; \
+                          print(True) if os.getenv('CI') else \
```

Comment:
Another thing: check if Gitlab defines that variable. If not, this might fail on our Gitlab instance.

Reply:
It's used within both:
Travis: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
Gitlab: https://docs.gitlab.com/ee/ci/variables/predefined_variables.html

Was there any other CI that Azul is tested with?

Reply:
Just those two.